i18n/en.json

```diff
@@ -0,0 +1,16 @@
+{
+  "label": "Blog article",
+  "pluralLabel": "Blog articles",
+  "releaseDate": "Publication Date",
```

Comment:
"Release Date"

Do not use the word publish anywhere

Reply:
Fixed